OpenDreamRuntime/DreamConnection.cs

```diff
@@ -96,6 +96,8 @@ public string SelectedStatPanel {
     public DreamConnection() {
         IoCManager.InjectDependencies(this);
 
+        _selectedStatPanel = "";
```

Comment:
An unmodified `/client.statpanel` var is `null` on byond. This makes the default an empty string on OD.